src/main/java/btdex/ui/SocialButton.java

```diff
@@ -11,22 +11,16 @@
 
 import javax.swing.JButton;
 
-import btdex.core.BurstNode;
-import btdex.core.ContractState;
-import btdex.core.Contracts;
-import btdex.core.Globals;
-import btdex.core.Market;
-import btdex.core.Markets;
+import btdex.core.*;
 import btdex.locale.Translation;
 import burst.kit.entity.response.AssetOrder;
+import jiconfont.IconCode;
 import jiconfont.icons.font_awesome.FontAwesomeBrands;
 import jiconfont.swing.IconFontSwing;
 
 public class SocialButton extends JButton implements ActionListener {
 	private static final long serialVersionUID = -7670367558338741748L;
-	
-	private static final String TWITTER_URL = "https://twitter.com/intent/tweet?text=";
```

Comment:
I'm don't think moving this out is good. No other class needs this and if you want to fix or improve something you need to check 2 different files.